### ISSUE TITLE
Placeholder for visualization

### ISSUE THREAD

**Issue by huulbaek** [pol-is/issues#85]
It would be nice with a placeholder for the visualization if there are no visualization present - for example, when there are not enough comments or votes yet.

It could be an image of the coordinate system with a text saying why the visualization is not ready yet.

**Comment by patcon** [comment]
Never thought of that, but I could totally see how ti would help people better plan how to integrate the embed into their own designs!

**Comment by huulbaek** [comment]
Yes, I didn't much think of it in terms of design - but I agree :)

**Comment by colinmegill** [comment]
Agree. This would be great.
